tests/product/test_all_products.py

```diff
@@ -14,6 +14,7 @@
     name="Description of the tested functionality",
 )
 class TestAllProducts:
+    @pytest.mark.skip(reason="Need to fix the bug in the test with lengths comparison")
```

Comment:
We need a task, otherwise no one will understand who is doing it

Reply:
Good. I'll take this task.